<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>456</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KeyBoardSimulator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="StatusGroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>331</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Status</string>
    </property>
    <widget class="QPushButton" name="ConnectButton">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
    <widget class="QComboBox" name="ComComboBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="DisconnectButton">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>20</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Disconnect</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="ManualModeGroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>441</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>Manual Mode</string>
    </property>
    <widget class="QPushButton" name="Key1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>75</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Key4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>75</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>4</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Key7">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>75</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>7</string>
     </property>
    </widget>
    <widget class="QPushButton" name="KeyStar">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>200</y>
       <width>75</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>*</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Key2">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>20</y>
       <width>75</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Key5">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>80</y>
       <width>75</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Key8">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>140</y>
       <width>75</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>8</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Key0">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>200</y>
       <width>75</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Key3">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>20</y>
       <width>75</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Key6">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>80</y>
       <width>75</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>6</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Key9">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>140</y>
       <width>75</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>9</string>
     </property>
    </widget>
    <widget class="QPushButton" name="KeySharp">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>200</y>
       <width>75</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>#</string>
     </property>
    </widget>
    <widget class="QPushButton" name="KeyBackspace">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>20</y>
       <width>75</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>Backspace</string>
     </property>
    </widget>
    <widget class="QPushButton" name="KeyUp">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>80</y>
       <width>75</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>Up</string>
     </property>
    </widget>
    <widget class="QPushButton" name="KeyDown">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>140</y>
       <width>75</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>Down</string>
     </property>
    </widget>
    <widget class="QPushButton" name="KeyOK">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>200</y>
       <width>75</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="WorkingModeGroupBox">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Working Mode</string>
    </property>
    <widget class="QComboBox" name="WorkingModeComboBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="AutoModeGroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>441</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Auto Mode</string>
    </property>
    <widget class="QLabel" name="PathLable">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Path:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="DirectoryComboBox">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>301</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QToolButton" name="PathButton">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>20</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QToolButton" name="OuchButton">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>50</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Ouch</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="PhoneLineEdit">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>301</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>11</number>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Phone:</string>
     </property>
    </widget>
   </widget>
   <zorder>ManualModeGroupBox</zorder>
   <zorder>StatusGroupBox</zorder>
   <zorder>WorkingModeGroupBox</zorder>
   <zorder>AutoModeGroupBox</zorder>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
